indico/modules/events/papers/templates/management/assignment.html

```diff
@@ -0,0 +1,95 @@
+{% extends 'layout/full_content_base.html' %}
+{% from 'events/papers/management/_paper_assignment_list.html' import render_paper_assignment_list %}
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
+
+{% block title %}
+    {% trans %}Paper Assignment{% endtrans %}
+{% endblock %}
+
+{% block description %}
+    {%- trans -%}
+        Assign <strong>judges</strong>, <strong>content reviewers</strong> and <strong>layout reviewers</strong>
+        to contributions in order to allow assessing its paper revisions.
+    {%- endtrans -%}
+{% endblock %}
+
+{% block content %}
+    <div class="toolbars space-after">
+        <div class="toolbar">
+            <div class="group">
+                <a class="i-button icon-checkbox-checked arrow js-dropdown" data-toggle="dropdown"></a>
+                <ul class="dropdown">
+                    <li>
+                        <a href="#" data-select-all="#assignment-list input:checkbox">{% trans %}All{% endtrans %}</a>
```

Comment:
Selecting a row doesn't highlight it. Make sure the `handleRowSelection` method is executed (is should be executed by calling `setupListGenerator` as we do in the `setupContributionList` function of the contribution list (`contributions/templates/management/contributions.html`)